Garima_Solihullsfc

Garima2412 opened the pull request:
Please review and provide feedback

Garima2412 committed:
Test

Solihullsfc

narendrashakya commented on Garima_Solihullsfc/Resource/Pages/HomePage.robot:
```diff
@@ -0,0 +1,48 @@
+#        ${languageDDUpdated}=       Replace string      ${languageDDUpdated}    <Language>      ${CurrentLanguage}
+#        wait until element is visible       ${languageDDUpdated}        timeout=30
+        ${PageTitle}=       Get Title
+        log to console                                  ${PageTitle}
```
It only printing the "English" Title on each iteration. Need to impliment wait and print the title.
**Title getting print on each Title :** Solihull Sixth Form College: Post 16 education, A Levels and BTECs — Solihull Sixth Form College

![image](https://user-images.githubusercontent.com/51281987/123214240-960d0600-d4e4-11eb-81f4-2aaddf93c357.png)

Garima2412 replied:
Updated

Garima2412 replied:
Updated - working now 
![result](https://user-images.githubusercontent.com/79889760/123233367-a24e8e80-d4f7-11eb-91d7-cadaae7e37c5.JPG)

mssquaretech replied:
Wow.. keep it up

narendrashakya commented on Garima_Solihullsfc/solihullsfc.robot:
```diff
@@ -0,0 +1,14 @@
+*** Settings ***
```
**Test** directory is not present, feature file need to be present in Test Directory.

narendrashakya reviewed (changes requested):
"Test" folder is not present

Garima2412 committed:
Updated

Changes are made in HomePage- Script also feature file is moved with the TEST Dir

narendrashakya commented on Configuration/Env.robot:
```diff
@@ -0,0 +1,15 @@
+*** Settings ***
+Library                     OperatingSystem
+Resource                    ../Pages/Adhoc.robot
```
Adhoc robot file is under **Resource** directory, resource path missing this directory. It giving below error.

Garima_Solihullsfc\Configuration\Env.robot' on line 3: Resource file '..\Pages\Adhoc.robot' does not exist.
**Correct path will be :** ../Resource/Pages/Adhoc.robot

narendrashakya reviewed (changes requested):
Please update the correct path under ENV robot file.

Garima2412 committed:
Update Env.robot

Updated the resource path

Garima2412 committed:
Update Env.robot

narendrashakya reviewed (approved):
Approved 👍🏼

mssquaretech reviewed (changes requested):
This branch contains multiple folders like Configuration, Resource, Test and Garima_Solihullsfc.
Expected : only one folder should be there - Garima_Solihullsfc.
Please remove other folders. 
Rest all is fine

Garima2412 committed:
Removed Extra dir

updated as per Comments

mssquaretech reviewed (commented):
Changes done.. ready to merge
